src/components/Instance/index.svelte

```diff
@@ -45,6 +45,14 @@
 
   <Types href="{id}" {types} definition="{expanded_def}" />
 
+  {#if examples}
+    <Examples {examples} href="{`${href}/examples`}" />
+  {/if}
+
+  {#if enumeration}
```

Comment:
I think it makes more sense to have the 'enum' block appears before the types, in term of document flow.